### PR TITLE
add basic Vim key binds

### PR DESCRIPTION
**Pull request by jamie-owen**
I maintain a couple of tools using `cliclack` and it would be great to satisfy my muscle memory by adding these key binds, let me know what you think.

### CODE REVIEW

**Comment by fadeevab** [comment]
@jamie-owen Well, no cons, right? It might be a way to go.

**Comment by jamie-owen** [comment]
Great, feel free to merge as in that case 👍

File: src/confirm.rs [hunk 1]
```diff
@@ -48,7 +48,12 @@ impl PromptInteraction<bool> for Confirm {
         let Event::Key(key) = event;
 
         match key {
-            Key::ArrowDown | Key::ArrowRight | Key::ArrowUp | Key::ArrowLeft => {
+            Key::ArrowDown
+            | Key::ArrowRight
+            | Key::ArrowUp
+            | Key::ArrowLeft
+            | Key::Char('h')
```
Comment (fadeevab) [inline src/confirm.rs:55]: Just realized, 'k' and 'j' also should be here

File: src/multiselect.rs [hunk 2]
```diff
@@ -125,12 +125,12 @@ impl<T: Clone> PromptInteraction<Vec<T>> for MultiSelect<T> {
...
 
         match key {
-            Key::ArrowLeft | Key::ArrowUp => {
+            Key::ArrowLeft | Key::ArrowUp | Key::Char('k') => {
                 if self.cursor > 0 {
                     self.cursor -= 1;
                 }
             }
-            Key::ArrowRight | Key::ArrowDown => {
+            Key::ArrowRight | Key::ArrowDown | Key::Char('j') => {
```
Comment (fadeevab) [inline src/multiselect.rs:133]: Both right and down vim keys should be here as well

File: src/select.rs [hunk 3]
```diff
@@ -114,12 +114,12 @@ impl<T: Clone> PromptInteraction<T> for Select<T> {
         }
 
         match key {
-            Key::ArrowUp | Key::ArrowLeft => {
+            Key::ArrowUp | Key::ArrowLeft | Key::Char('k') => {
```
Comment (fadeevab) [inline src/select.rs:117]: Lastly, you have to test it with filter mode on. Letters will move the cursor while entering into the filter field.

Comment (jamie-owen) [reply src/select.rs]: With `.filter_mode()` on the Vim keybinds don't work for moevement, I think this is preferable to not being able to filter by `hjkl`.

Comment (fadeevab) [reply src/select.rs]: @jamie-owen Yes, I mean it's not preferable, so it lgtm

**Comment by fadeevab** [comment]
@jamie-owen published in `v0.3.5`, thanks!

**Pull request merged**